.github/workflows/pullrequest.yml

```diff
@@ -99,7 +99,9 @@ jobs:
         continue-on-error: true
         uses: ./.github/actions/cache
         with:
-          path: node_modules
+          path: |
```

Comment:
Hash is still only calculated for the root of the mono repo. This will break cache so it only works if root is changed.

Not 100% if the custom cache supports array - messing with cache can break all build, deployments etc.

This creates huge cache for everyone – fetching node_modules for native app for every task.